go/README.md

```diff
@@ -118,3 +118,57 @@ func main() {
     }
 }
 ```
+### Custom Context
+
+If you want even greater control over the lifecycle of the HTTP requests consider providing a [context](https://pkg.go.dev/context). Contexts can be set for all requests or per request. They can be combined with the timeout options mentioned in the previous section to fine-tune the lifecycle of your requests.
+
+If you wish to include timeout functionality in your custom context then you should leverage [context.WithTimeout](https://pkg.go.dev/context#WithTimeout).
```

Comment:
Also add: ""The SDK will ignore any timeout described in the previous Timeout section for all requests if you use custom context for all requests, or per requests if you use custom context per request."

Reply:
See my previous comment - this is inaccurate.